tutorials/migration-guide.html

```diff
@@ -0,0 +1,530 @@
+          <code>afterColumnMove</code> have different set of parameters now.
+      </p>
+
+      <p>
+          Methods <code>moveColumn</code>, <code>moveColumns</code>, <code>moveRow</code>, and <code>moveRows</code>
+          have their arguments changed. From this version onward, the <code>target</code> argument was changed to
+          <code>finalIndex</code> argument. They work differently now and a <strong>new <code>dragRow</code>,
+          <code>dragRows</code>, <code>dragColumn</code> and <code>dragColumns</code> methods overtook old methods
+          place in terms of functionality</strong>. To put it bluntly, if you want to preserve previous functionality of
+          <code>moveRows</code> rename it to the <code>dragRows</code>.
+      </p>
+
+      <p>
+          <strong>Important note!</strong> If <code>NestedRows</code> is set to <code>enabled</code> the move is
+          possible only by the drag in the UI or <code>dragRow</code>/<code>dragRows</code> methods of the
+          <code>ManualRowMove</code> plugin.
+      </p>
+
+      <p>
+          The "drag" methods comes with the <code>dropIndex</code> parameter. It directs where to
+          <strong>place</strong> the dragged elements. The place you intend to drag the element is managed by
+          <strong>drop indexes</strong>. You can imagine some sort of a drop zone between actual indexes of elements:
+      </p>
+
+      <img src="images/drag_action.svg" alt="drag_action" />
+
+      <p>The "move" methods comes with the <code>finalIndex</code> parameter. It tells where to <strong>overlap</strong> the first element from the moved ones. The place you intend to move the element is managed by <strong>visual indexes</strong>.</p>
+
+      <img src="images/move_action.svg" alt="move_action" />
+
+      <p>
+          Please note that in case of "move" methods some move actions are limited. For example, if you initiate a move
+          of <strong>more than one element</strong> to the <strong>last position</strong> (visual index = the number of items - 1)
+          the operation will be canceled. The first element in the collection you would like to move will try to reach
+          the last position (<code>finalIndex</code>) which is feasible. However, the next ones will attempt to reach
+          the position exceeding the number of all items.
+      </p>
+
+      <p>
+          You can find the plugin's isMovePossible API method useful when you want to determine if the move action is
+          possible. <code>movePossible</code> parameter of <code>beforeRowMove</code>, <code>afterRowMove</code>,
+          <code>beforeColumnMove</code>, and <code>afterColumnMove</code> hooks may be helpful as well.
+      </p>
+
+      <h3>Changes in ManualColumnFreeze plugin behavior</h3>
+
+      <p>
+          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the <strong>8.0.0v</strong>
```

Comment:
```suggestion
          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the <strong>v8.0.0</strong>
```
I think that the `v` should be placed as a prefix. It looks more natural.

Reply:
```suggestion
          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the 8.0.0
```
Or this. As the migration guide uses mostly simplified notation (without prefix and using regular font style) so it's more consistent with the overall document. Unless the emphasizing of the version, in this case, was intentional.